packages/appkit/src/client.ts

```diff
@@ -490,6 +496,14 @@ export class AppKit {
     AccountController.setProfileImage(profileImage, chain)
   }
 
+  public setEmail: (typeof AccountController)['setEmail'] = (email, chain) => {
+    AccountController.setEmail(email, chain)
+  }
+
+  public setUsername: (typeof AccountController)['setUsername'] = (username, chain) => {
+    AccountController.setUsername(username, chain)
```

Comment:
this is only for social accounts right? wouldn't it make sense to store the user object directly?